### ISSUE TITLE
"Minimum" Callout not triggert @ DH

### ISSUE THREAD

**Issue by andrejetcat** [flybywiresim/a32nx#6614]
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2022-01-29T14:15:42+00:00",
  "ref": "experimental",
  "sha": "aa4fd10e6bd4a283c4e19e2635ddcc35f4922228",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

I made 4 flight today and on all approaches the "minimum" callout do not triggert @ minimum.
(I set it properly on the appr-page) 

### Expected behavior

Minimum - callout should audible @ minimum (DH) 

### Steps to reproduce

1. make a approach on any runway 
2. set respective Minumum
3. wait for the Minimum - callout

**Comment by beheh** [comment]
What type of minimum did you set? BARO or RADIO?

**Comment by andrejetcat** [comment]
I set "BARO" (510ft @ EDDK RWY 32R), also tryes with 590ft for RWY24 @EDDK.

**Comment by beheh** [comment]
Alright. Maybe this is related to the sound, but in any case this will be resolved in the FWC branch once that lands.

**Comment by andrejetcat** [comment]
awesome, thank you for your support!!! :)

**Comment by donstim** [comment]
This must have changed very recently on the experimental version. I flew yesterday and am sure I had a minimums callout, but I can confirm it doesn't work now.

**Comment by donstim** [comment]
I had the "100 above" and the minimums callout on my flight into KFLL today.

**Comment by 2hwk** [comment]
Is this still an issue? Otherwise this will be closed for staleness

**Comment by juliansebline** [comment]
@2hwk Beheh did a PR on it

**Comment by beheh** [comment]
This is no longer relevant as this was an Exp-only bug that has since been fixed (by #7610).
